.github/workflows/release-embedding-sdk.yml

```diff
@@ -360,17 +361,11 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
 
-      - uses: actions/create-github-app-token@v1
-        id: app-token
-        with:
-          app-id: ${{ secrets.METABASE_BOT_APP_ID }}
-          private-key: ${{ secrets.METABASE_BOT_APP_PRIVATE_KEY }}
-
       - name: Edit PR adding useful information (using Metabase bot app)
         run: |
           gh pr edit --add-reviewer @metabase/embedding,albertoperdomo --add-label no-backport
         env:
-          GH_TOKEN: ${{ steps.app-token.outputs.token }}
+          GH_TOKEN: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
```

Comment:
This is just a clean up. Previously, we used a generated token. Since `GITHUB_TOKEN` wasn't enough to add reviewers to a PR.

But we already have `METABASE_AUTOMATION_USER_TOKEN` which is a Personal Access Token with proper permissions.